### PR TITLE
Add document for tiup-cluster

### PR DESCRIPTION
**Pull request by lucklove**
Signed-off-by: lucklove <gnu.crazier@gmail.com>

### How to contact you? 
- Email:
- Telephone:
- WeChat:
- Others:

### CODE REVIEW

**Review by july2993 — APPROVED** [review]

File: session2/chapter1/tiup-cluster.md [hunk 1]
```diff
@@ -0,0 +1,265 @@
+### 1.1.3 用 TiUP 部署生产集群
+
+上一节介绍了如何使用 TiUP 结合组件 `playground` 快速启动一个本地集群，这样的集群可以用来本地测试，但该集群显然不能用于生产环境。因此我们推出了用于快速部署生产环境的 `tiup-cluster` 组件，该组件像 `playground` 部署本地集群一样快速部署生产集群，对比 `playground`，它提供了更强大的集群管理功能，包括对集群的升级，缩容，扩容甚至操作审计等。下面我们一起看看这个小工具的玩法。
```
Comment (lonng) [inline session2/chapter1/tiup-cluster.md]:
```suggestion
上一节介绍了如何使用 TiUP 结合组件 `playground` 快速启动一个本地集群，这样的集群可以用来本地测试，但该集群显然不能用于生产环境。因此我们推出了用于快速部署生产环境的 [`cluster`](https://github.com/pingcap-incubator/tiup-cluster) 组件，该组件像 `playground` 部署本地集群一样快速部署生产集群，对比 `playground`，它提供了更强大的集群管理功能，包括对集群的升级，缩容，扩容甚至操作审计等。下面我们一起看看这个组件的用法。
```

File: session2/chapter1/tiup-cluster.md [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+  - host: 172.16.5.139
+  - host: 172.16.5.140
+
+grafana_servers:
+  - host: 172.16.5.134
+
+monitoring_servers:
+  - host: 172.16.5.134
+```
+将该文件保存为 `/tmp/topology.yaml`。假如我们想要使用 TiDB 的 `v3.0.1` 版本，集群名字命名为 `prod-cluster`，则执行:
```
Comment (lonng) [inline session2/chapter1/tiup-cluster.md]:
```suggestion
将该文件保存为 `/tmp/topology.yaml`。假如我们想要使用 TiDB 的 `v4.0.0-rc` 版本，集群名字命名为 `prod-cluster`，则执行:
```
尽量往新版本引导

File: session2/chapter1/tiup-cluster.md [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+```
+tiup cluster deploy prod-cluster v3.0.1 /tmp/topology.yaml
```
Comment (lonng) [inline session2/chapter1/tiup-cluster.md]: ditto

File: session2/chapter1/tiup-cluster.md [hunk 4]
```diff
@@ -0,0 +1,265 @@
...
+prometheus  172.16.5.134  9090         deploy/prometheus-9090,data/prometheus-9090
+grafana     172.16.5.134  3000         deploy/grafana-3000
+Attention:
+    1. If the topology is not what you expected, check your yaml file.
+    1. Please confirm there is no port/directory conflicts in same host.
+Do you want to continue? [y/N]:
+```
+输入密码后 `tiup-cluster` 便会下载需要的组件并部署到对应的机器上，当看到以下提示时说明部署成功：
+```
+Deployed cluster `prod-cluster` successfully
```
Comment (lonng) [inline session2/chapter1/tiup-cluster.md:118]: 这个已经删除了，https://github.com/pingcap-incubator/tiup-cluster/pull/239，新的版本应该不会存在了

File: session2/chapter1/tiup-cluster.md [hunk 5]
```diff
@@ -0,0 +1,285 @@
...
+[root@localhost ~]# tiup cluster list
+Starting /root/.tiup/components/cluster/v0.4.5/cluster list
+Name          User  Version    Path                                               PrivateKey
+----          ----  -------    ----                                               ----------
+prod-cluster  tidb  v3.0.12    /root/.tiup/storage/cluster/clusters/prod-cluster  /root/.tiup/storage/cluster/clusters/prod-cluster/ssh/id_rsa
+```
+
+#### 4. 启动集群
+
+如果在上一步中没有立刻启动集群，我们可以执行命令将该集群启动起来，如果忘记了已经部署的集群的名字，可以使用 `tiup cluster list` 查看，启动集群的命令：
```
Comment (kissmydb) [inline session2/chapter1/tiup-cluster.md]:
```suggestion
上一步部署成功后，我们可以执行命令将该集群启动起来，如果忘记了已经部署的集群的名字，可以使用 `tiup cluster list` 查看，启动集群的命令：
```

**Review by kissmydb — APPROVED** [review]
LGTM

**Pull request merged**